tests/test_config.py

```diff
@@ -51,6 +52,9 @@ def test_parses_all_keys(self, project_id, credentials, custom_encoder):
         assert config.credentials == credentials
         assert config.middleware == ["rele.contrib.DjangoDBMiddleware"]
 
+    @patch.dict(
+        os.environ, {},
```

Comment:
It seems like travis has a value returned for `credentials, project = get_google_defaults()`, what you might have to do is mock this function.

Reply:
[It is indeed a Travis envvar showing up](https://github.com/travis-ci/terraform-config/blob/master/gce-production-1/main.tf#L14)